draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1026,17 +1033,17 @@ new_phase: CarefulResumePhase,
 state_data: CarefulResumeStateParameters,
 ? restored_data: CarefulResumeRestoredParameters,
 ? trigger:
-        ; for the Safe Retreat phase
-        "packet_loss" /
-        ; for the Unvalidated phase
+        ; for the Unvalidated phase, when no unvalidated packets
         "congestion_window_limited" /
         ; for the Validating phase
         "first_unvalidated_packet_acknowledged" /
-	; for the Normal phase
+	; for the Normal phase, and no remaining unvalidated packets to be acknowledged
```

Comment:
maybe better as:
 for the Normal phase, when no remaining unvalidated packets need to be acknowledged